datasets/mslr2022/README.md

```diff
@@ -0,0 +1,179 @@
+    - [Dataset Summary](#dataset-summary)
+    - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+    - [Languages](#languages)
+  - [Dataset Structure](#dataset-structure)
+    - [Data Instances](#data-instances)
+    - [Data Fields](#data-fields)
+    - [Data Splits](#data-splits)
+  - [Dataset Creation](#dataset-creation)
+    - [Curation Rationale](#curation-rationale)
+    - [Source Data](#source-data)
+      - [Initial Data Collection and Normalization](#initial-data-collection-and-normalization)
+      - [Who are the source language producers?](#who-are-the-source-language-producers)
+    - [Annotations](#annotations)
+      - [Annotation process](#annotation-process)
+      - [Who are the annotators?](#who-are-the-annotators)
+    - [Personal and Sensitive Information](#personal-and-sensitive-information)
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/allenai/mslr-shared-task
+- **Repository:** https://github.com/allenai/mslr-shared-task
+- **Paper:** https://aclanthology.org/2021.emnlp-main.594
+- **Leaderboard:** https://github.com/allenai/mslr-shared-task#leaderboard
+- **Point of Contact:** https://github.com/allenai/mslr-shared-task#contact-us
+
+### Dataset Summary
+
+The Multidocument Summarization for Literature Review (MSLR) Shared Task aims to study how medical evidence from different clinical studies are summarized in literature reviews. Reviews provide the highest quality of evidence for clinical care, but are expensive to produce manually. (Semi-)automation via NLP may facilitate faster evidence synthesis without sacrificing rigor. The MSLR shared task uses two datasets to assess the current state of multidocument summarization for this task, and to encourage the development of modeling contributions, scaffolding tasks, methods for model interpretability, and improved automated evaluation methods in this domain.
+
+### Supported Tasks and Leaderboards
+
+This dataset is used for the MSLR2022 Shared Task. For information on the shared task leaderboard, please refer [here](https://github.com/allenai/mslr-shared-task#leaderboard).
+
+### Languages
+
+English
+
+## Dataset Structure
+
+More information on dataset structure [here](https://github.com/allenai/mslr-shared-task#data-structure).
+
+### Data Instances
+
+[Needs More Information]
+
+### Data Fields
+
+[Needs More Information]
+
+### Data Splits
+
+[Needs More Information]
+
+## Dataset Creation
+
+Please refer to the following papers for details about dataset curation:
```

Comment:
(optional) maybe you can summarize the dataset curation process in this section ?